### ISSUE TITLE
Add arguments at startup

### ISSUE THREAD

**Issue by akky5112** [xupefei/Locale-Emulator#449]
I'm sorry for my bad English.

I want to start .exe with arguments. Example: origin.exe -AutoStart
Would you please add this option?

**Comment by xupefei** [comment]
Right-click on the executable and choose "Modify Application Profile", then there is a text box for arguments at the top of the window.
